src/main/java/org/mskcc/cmo/metadb/service/impl/LabelGenMessageHandlingServiceImpl.java

```diff
@@ -146,7 +146,7 @@ public void run() {
                         Map<String, List<SampleMetadata>> patientSamplesMap = getPatientSamplesMap(samples);
 
                         // udpated samples list will store samples which had a label generated successfully
-                        List<SampleMetadata> updatedSamples = new ArrayList<>();
+                        List<Object> updatedSamples = new ArrayList<>();
```

Comment:
if not `Object` then request json is not parsed successfully by metadb/consistency checker